### ISSUE TITLE
CSS seems to make X image too low in FF, Chrome and Safari on Mac

### ISSUE THREAD

**Issue by OpenESignForms** [tjkaal/ResetButtonForTextField#8]
I'm using the 1.2.1 version on Vaadin 7.4.7 and noted that the X icon is below the input field.  I'm pretty sure this wasn't true before, but as we've upgraded versions, we don't always notice UI changes like this.

To correct it, we had to add this CSS to our theme to override -- essentially remove -- the vertical-align and top attributes shipped with the addon:

/* 6/30/2015: fix for resetbuttonfortextfield where X image is showing too low */
.resetbuttonfortextfield-resetbutton { vertical-align: baseline !important; top: auto !important; }

I've included an image that shows how it looks in Chrome without the CSS overrides.
![image](https://cloud.githubusercontent.com/assets/1279424/8440047/a558b5d6-1f24-11e5-8ebf-6e99f866280d.png)

Our theme is basically built on reindeer.  Is there perhaps some issue with changes people are making to use Valo theme?

**Comment by tjkaal** [comment]
You're absolutely correct, while refactoring the project to be based on Maven I also made [a really quick fix on CSS](https://github.com/tjkaal/ResetButtonForTextField/commit/717d7a5a031a801bc4976949b3335ba4c7e8a0bc#diff-9561c478e23ff8b105f1c69f6e08c346) because the button was not positioned correctly when Valo theme was used.

Didn't really think about other themes at that point. Currently, I suggest that you do just the thing you've been doing: fix the issue in your own theme.

Still, I think that the button should be aligned correctly regardless of the theme used and as soon as I have some time I'll take a look on this and see if this can be corrected so that the positioning would work universally.
